commons/src/main/java/com/pryv/interfaces/DeleteCallback.java

```diff
@@ -0,0 +1,10 @@
+package com.pryv.interfaces;
+
+import com.pryv.model.ApiResource;
+
+public interface DeleteCallback <T extends ApiResource>  {
+
+    void onSuccess(String successMessage, String deletedId, Double serverTime);
```

Comment:
Don't you want to retrieve the full Event when it has been trashed?

Reply:
Did you mean Access? The API just deletes and responds with the deleted id for Accesses

Reply:
No I meant Events.